packages/iris-grid/src/IrisGrid.jsx

```diff
@@ -1371,6 +1373,24 @@ export class IrisGrid extends Component {
     this.grid.forceUpdate();
   }
 
+  freezeColumnByColumnName(columnName) {
+    const { frozenColumns } = this.state;
+    log.debug2('freezing column', columnName);
```

Comment:
If `frozenColumns` is null here, then you need to pull the frozen columns from the layoutHints in the model then add/remove from that array and set it